pkg/operator/targetconfigcontroller/targetconfigcontroller.go

```diff
@@ -126,6 +126,7 @@ func (c TargetConfigController) sync() error {
 	}
 	requiredPaths := [][]string{
 		{"serviceAccountPublicKeyFiles"},
+		{"servingInfo", "namedCertificates"},
```

Comment:
this break upgrade. My cluster does not have this field.

Reply:
> this breaks upgrade. My cluster does not have this field.

The control loop for configobservation will add it as soon as it runs.